### DISCUSSION TITLE
자료구조/심화 블록

### DISCUSSION THREAD

**Discussion by yeahhhhj** [2021community/Algorithm-skills#40] [Q&A]
자료구조 심화 내용에 어떤걸 더 넣으면 좋을까요? 😢
-------------
* 트리
  *  이진 트리
  * 이진 탐색 트리
  * AVL 트리
  * 힙

* 해싱

* 그래프

**Comment by alirz-pixel** [comment]
트리에서 이진 트리랑 이진 탐색 트리는 비슷한 자료 구조라서 두 개를 이진 트리로 합치고, 스패닝 트리 넣는 건 어떠신가요?

그리고
그래프 블록 하위에 '방향성  비순환 그래프 (DAG)', '방향 그래프', '강한 결합 요소 (scc)' 추가하면 좋을거 같네요!

**Reply by yeahhhhj** [reply]
감사합니닷!!

**Comment by alirz-pixel** [comment]
아! 그리고 해싱보다는 해시가 더 좋을거 같고, 해시 하위블록에 '딕셔너리, 맵' 을 넣는것도 나쁘지 않아보이네요!

**Reply by yeahhhhj** [reply]
넵 참고하겠습니다~

**Comment by Zzangeun** [comment]
이진트리에 트라이 트리도 넣으면 좋을 거 같아요

**Comment by Zzangeun** [comment]
우선순위 큐도 넣으면 좋을것 같네요~
